bigbluebutton-html5/imports/ui/components/whiteboard/shapes/helpers.js

```diff
@@ -0,0 +1,51 @@
+// applies zooming to the stroke thickness
+const zoomStroke = (thickness, widthRatio, heightRatio) => {
+  let ratio;
+
+  ratio = (widthRatio  + heightRatio) / 2;
+  return thickness * 100 / ratio;
+};
+
+const formatColor = (color) => {
+  //let color = this.props.shape.shape.shape.color;
+  if (color == null) {
```

Comment:
@kepstin , double-checked it.
Color is always a number. 0 if it's black and different integer values in the other cases.
And 0 is always false here, no matter if we use `==` or `===`.